pkg/cmd/root.go

```diff
@@ -57,7 +57,7 @@ func Execute() (*types.CmdValues, error) {
 
 func addProviderFlags(cmd *cobra.Command, vals *types.CmdValues) *cobra.Command {
 
-	cmd.Flags().StringSliceVarP(&vals.Files, "files", "f", []string{"docker-compose.yml"}, "Provide docker-compose files, comma separated.")
+	cmd.Flags().StringSliceVarP(&vals.Files, "files", "f", nil, "Provide docker-compose files, comma separated.")
```

Comment:
But it does not show `docker-compose.yaml`, so I removed it from here.

I'd say we add to the usage field that currently says `Provide docker-compose files, comma separated` to say, `Provide docker-compose files, comma separated, takes docker-compose.yml or docker-compose.yaml if no file is specified`